### PR TITLE
Update the README.md file to describe the repo.

### PR DESCRIPTION
**Pull request by Affluentt**
Updated the README file and personalized it with my name.

### CODE REVIEW

File: walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/index.html [hunk 1]
```diff
@@ -8,10 +8,10 @@
   <body>
     <div id="content">
       <h1>Stanley 🐈</h1>
-      <p>This is Stanley.</p>
+      <p>This is Stanley. He is awesome and smrt. </p>
```
Comment (thechungster) [inline walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/index.html]: Small typo: smart

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -2,8 +2,14 @@
   margin-left: auto;
   margin-right: auto;
   width: 650px;
+  font-family: sans-serif;
 }
-
+ 
+#p {
```
Comment (thechungster) [inline portfolio/src/main/webapp/style.css]: Is this supposed to be just `p {`?
I don't see any elements with id of p in the html
To select all p elements, `p {` will work
See https://www.w3schools.com/css/css_selectors.asp if you would like to know more about selectors

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -9,11 +9,54 @@
   <body>
     <div id="content">
       <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+ 
+      <h3>Background</h3>
+      <p>Hello, my name is Kevin Alfaro and this is my portfolio!</p>
+      <p>I currently am attending the University of Washington. I am a junior
```
Comment (thechungster) [inline portfolio/src/main/webapp/index.html]: nit: if the text spans multiple lines, I prefer to have a line break so it's easier to read
```
<p>
  Your text content
  that spans multiple lines
</p>
```
I see that some of the text below follows this. I would say choose one style and be consistent

**Review by thechungster — CHANGES_REQUESTED** [review]

File: walkthroughs/week-2-server/examples/page-view-counter/src/main/java/com/google/sps/servlets/PageViewServlet.java [hunk 4]
```diff
@@ -24,7 +24,11 @@
 @WebServlet("/page-views")
 public class PageViewServlet extends HttpServlet {
 
-  private int pageViews = 0;
+    /**
```
Comment (thechungster) [inline walkthroughs/week-2-server/examples/page-view-counter/src/main/java/com/google/sps/servlets/PageViewServlet.java]: The comment block doesn't seem to be used, remove it? or add a comment if that's what was intended

File: portfolio/src/main/webapp/style.css [hunk 5]
```diff
@@ -2,8 +2,17 @@
   margin-left: auto;
   margin-right: auto;
   width: 650px;
+  font-family: sans-serif;
+}
+ 
+p {
```
Comment (thechungster) [inline portfolio/src/main/webapp/style.css]: To select `p` and `li` you can do it in one rule

```
p, li {
    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
}
```

**Review by thechungster — CHANGES_REQUESTED** [review]
Good Job!

File: walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/index.html (hunk 1 above)
Comment (Affluentt) [reply walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/index.html]: Done!

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (Affluentt) [reply portfolio/src/main/webapp/style.css]: Done!

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (Affluentt) [reply portfolio/src/main/webapp/index.html]: Done!

File: portfolio/src/main/webapp/style.css (hunk 5 above)
Comment (Affluentt) [reply portfolio/src/main/webapp/style.css]: Done!

**Review by thechungster — APPROVED** [review]

**Pull request merged**